docker-compose.yml

```diff
@@ -27,4 +27,4 @@ services:
     ports:
       - "6060:8080"
     volumes:
-      - ${VEP_CACHE}:/opt/vep/.vep/:ro
+      - ${VEP_CACHE}:/opt/vep/.vep/
```

Comment:
Need to remove "read-only", otherwise will have error:
```
java.nio.file.FileSystemException: /opt/vep/.vep/tmp: Read-only file system

at sun.nio.fs.UnixException.translateToIOException(UnixException.java:91)

at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:102)

at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:107)

at sun.nio.fs.UnixFileSystemProvider.createDirectory(UnixFileSystemProvider.java:384)

at java.nio.file.Files.createDirectory(Files.java:674)

at org.genomenexus.vep_wrapper.VepRunner.createTmpDirIfNecessary(VepRunner.java:62)

at org.genomenexus.vep_wrapper.VepRunner.run(VepRunner.java:119)

at org.genomenexus.vep_wrapper.VepController.getVepAnnotation(VepController.java:47)
```